packages/styling/src/interfaces/IEffects.ts

```diff
@@ -0,0 +1,19 @@
+import { IRawStyle } from '@uifabric/merge-styles';
+
+/**
+ * @internal
+ * Experimental interface for decorative styling in a theme.
+ */
+export interface IEffects {
+  elevation4: IRawStyle;
+  elevation8: IRawStyle;
+  elevation16: IRawStyle;
+  elevation64: IRawStyle;
+
+  roundedCorner2: number;
```

Comment:
I made this a number so you can do math on it. (For example, there's a really big button so you want double the border-radius.) Is that a useful use case or should I just make it a string? Having it as a number also limits it to pixel values only... which generally makes sense. You probably never want to have a % border-radius value globally.

Reply:
How will this interface grow over time? Per David's example, I'm a bit concerned that if `IEffects` applies to a `shadow` style variable, for example, that typings would allow `shadow: roundedCorner2` without necessarily making sense. Does this type need to be broken down more?

Reply:
That wouldn't be possible since a number isn't a valid type for box-shadow. You could, however, use roundedCorner2 for border-width or other random things. I think that's fine, since we might use font size for a padding value somewhere, to make room for text that will be rendered later, or to align text with another element.

Anyways, that's part of my motivation for making the elevation styles IRawStyles, since they would be so specific to box-shadow, that we might as well do that for the user. (Additionally, this lets you do fancy things like insert an image to use as the shadow, or mess with borders/outlines, or use a transform or glow effect, and so on.)